api/src/main/java/org/openmrs/module/metadatadeploy/bundle/CoreConstructors.java

```diff
@@ -56,6 +60,16 @@
 public class CoreConstructors {
 
 	/**
+	 * Constructs a dbsessionFactory source
+	 * @return the transient object
+	 */
+	public static DbSessionFactory dbSessionFactory(SessionFactory sessionFactory) {
```

Comment:
I read through the commit you sent me and i had done that earlier, without much success..
may be let me re-do and see it could be i missed out something..